packages/spark-core/src/lib/spark-plugin.js

```diff
@@ -6,9 +6,22 @@
 
 import AmpState from 'ampersand-state';
 import util from 'util';
+import {makeSparkPluginStore} from './storage';
 
 const SparkPlugin = AmpState.extend({
   derived: {
+    boundedStorage: {
+      deps: [],
+      fn() {
+        return makeSparkPluginStore(`bounded`, this);
+      }
+    },
+    unboundedStorage: {
```

Comment:
It's interesting to have derived attributes that aren't dependent on anything. Is there a reason they aren't session attributes?